docs/dataservice-write-example.md

```diff
@@ -34,6 +34,39 @@ To execute the example, run:
 
 If everything is fine, you would see `Publish Successful - TraceID: <TraceId generated by the platform>`.
 
+### Building on Android
+
+Prerequsities: make sure that you have correctly setup the Android environment (downloaded both Android SDK and NDK from official site). Also, make sure that your provided correct keyId, keySecret, and other information stated in the beginning of this README file.
+
+This example shows how to integrate and use the OLP C++ EDGE SDK in Android project.
+
+First, configure the project with `EDGE_SDK_BUILD_EXAMPLES` set to `ON`, path to NDK's Android toolchain file set via `CMAKE_TOOLCHAIN_FILE`, and, optionally, the `ANDROID_` CMake variables, for example to configure SDK version - `ANDROID_PLATFORM`, or ABI for which project will be configured - `ANDROID_ABI`:
+
+```bash
+mkdir build && cd build
+cmake .. -DEDGE_SDK_BUILD_EXAMPLES=ON -DCMAKE_TOOLCHAIN_FILE=$NDK_ROOT/build/cmake/android.toolchain.cmake -DANDROID_ABI=arm64-v8a
+```
+
+Executing this `CMake` command will generate a `Gradle` project in the `build/examples/dataservice-write/android` folder. Before using it, you first must install the OLP C++ EDGE SDK libraries into your sysroot:
+
+```bash
+(sudo) make install
+```
+
+After running those steps, you can use `gradlew` to build and install the apk on your Android device:
```

Comment:
```suggestion
Now you can use `gradlew` to build and install the apk on your Android device:
```

Reply:
I changed a bit this paragraph considering your changes. Please, take a look.